scanner/scanner.go

```diff
@@ -0,0 +1,361 @@
+// Copyright © 2017 Microsoft <nitinsingla@microsoft.com>
```

Comment:
Minor: please remove the date 2017, for new files we don't need to put a date.

Reply:
sure got it Ze, Thanks